### PR TITLE
[FIX] Resolve single-space bug.

### LINKED ISSUE #16090: The messages are not respecting the spaces in between after sending
### Description:
On sending any message if we set multiple spaces before any word, the message renders as having just one space

### Steps to reproduce:

1. Write any message with a different number of spaces 
2. Send the message
3. Observe the message now has only one space between any two words

### Expected behavior:

It should have spaces as described by the user

### Actual behavior:

Selection_009

### PR DESCRIPTION
**Pull request by manosriram**
Closes #16090

Resolves a bug where only a single space is being rendered even when multiple spaces are given.

Before Fix

After Fix

### CODE REVIEW

File: app/ui-message/client/message.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pre {
+    font-family: -apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Helvetica Neue','Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol','Meiryo UI',Arial,sans-serif;
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

Comment (manosriram) [reply app/ui-message/client/message.css]: Please have a look now.

File: app/ui-message/client/message.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    font-family: -apple-system,
+                 BlinkMacSystemFont,
+                 'Segoe UI',
+                 Roboto,
+                 Oxygen,
+                 Ubuntu,
+                 Cantarell,
+                 'Helvetica Neue',
+                 'Apple Color Emoji',
+                 'Segoe UI Emoji',
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+                 'Segoe UI Symbol',
+                 'Meiryo UI',
+                 Arial,sans-serif;
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+                 'Segoe UI',
+                 Roboto,
+                 Oxygen,
+                 Ubuntu,
+                 Cantarell,
+                 'Helvetica Neue',
+                 'Apple Color Emoji',
+                 'Segoe UI Emoji',
+                 'Segoe UI Symbol',
+                 'Meiryo UI',
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
+pre {
+    font-family: -apple-system,
+                 BlinkMacSystemFont,
+                 'Segoe UI',
+                 Roboto,
+                 Oxygen,
+                 Ubuntu,
+                 Cantarell,
+                 'Helvetica Neue',
+                 'Apple Color Emoji',
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 6]
```diff
@@ -0,0 +1,15 @@
+pre {
+    font-family: -apple-system,
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+                 BlinkMacSystemFont,
+                 'Segoe UI',
+                 Roboto,
+                 Oxygen,
+                 Ubuntu,
+                 Cantarell,
+                 'Helvetica Neue',
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 8]
```diff
@@ -0,0 +1,15 @@
+pre {
+    font-family: -apple-system,
+                 BlinkMacSystemFont,
+                 'Segoe UI',
+                 Roboto,
+                 Oxygen,
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+                 Ubuntu,
+                 Cantarell,
+                 'Helvetica Neue',
+                 'Apple Color Emoji',
+                 'Segoe UI Emoji',
+                 'Segoe UI Symbol',
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 10]
```diff
@@ -0,0 +1,15 @@
+pre {
+    font-family: -apple-system,
+                 BlinkMacSystemFont,
+                 'Segoe UI',
+                 Roboto,
+                 Oxygen,
+                 Ubuntu,
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 11]
```diff
@@ -0,0 +1,15 @@
+pre {
+    font-family: -apple-system,
+                 BlinkMacSystemFont,
+                 'Segoe UI',
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 12]
```diff
@@ -0,0 +1,15 @@
+pre {
+    font-family: -apple-system,
+                 BlinkMacSystemFont,
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css (hunk 6 above)
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+                 'Segoe UI',
+                 Roboto,
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+                 Oxygen,
+                 Ubuntu,
+                 Cantarell,
```
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

File: app/ui-message/client/message.css (hunk 3 above)
Comment (sampaiodiego) [inline app/ui-message/client/message.css]: Codacy Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)

**Comment by ggazzo** [comment]
> it's working as we planned, if you check, github has the same behavior... you can use, text or text if you want spaces being preserved

**Pull request closed without merging**